### DISCUSSION TITLE
[Flutter Web] Run firebase_analytics in no consent mode

### DISCUSSION THREAD

**Discussion by razfazz** [firebase/flutterfire#10752] [Feature request]
Is it possible to run firebase_analytics in light mode that according to GDPR I am not obliged to display a cookie consent?

**Comment by zogol** [comment]
i would love to know that as well

**Comment by aleeshaford** [comment]
this is actually very important to know to be able to use it legally in europe. i will stick to the simplyanalytics project until this is cleared out
